.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,19 +19,15 @@ Fixes #<NUM> <!-- (e.g. Fixes #58.) -->
 
 ### What should a reviewer concentrate their feedback on?
 
-<!-- This section is particularly useful if you have a pull request that is
-still in development. You can guide the reviews to focus on the parts that are
-ready for their comments.
-We suggest using bullets (indicated by * or -) and filled checkboxes [x] here -->
+<!-- This section is particularly useful if you have a pull request that is still in development. You can guide the reviews to focus on the parts that are ready for their comments. We suggest using bullets (indicated by * or -) and filled checkboxes [x] here -->
 
 - [ ] *Lorem ipsum dolor sit amet, consectetur adipiscing.*
-- [ ] *Lorem ipsum dolor sit amet, consectetur adipiscing.*
+- [ ] Everything looks ok?
 
 
 ### Acknowledging contributors
 
-<!-- Please select the box when confirmed -->
-
-By submitting this pull request I confirm that:
+<!-- Please select the correct box -->
 
-- [ ] all contributors to this pull request who wish to be included are named in [contributors.md](https://github.com/alan-turing-institute/the-turing-way/blob/master/contributors.md).
+- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
+- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file: <!-- replace this text with the GitHub IDs of any new contributors -->
```

Comment:
This change will be really handy! I couldn't be bothered to figoure out how to add to a PR from the GitHub UI so I gave up and just checked the box anyway; this would help.

Reply:
Thanks @yochannah! In generally my strong preference is one sentence per line. But I really dislike the way that GitHub issues keep all these line breaks and don't wrap them as expected.

![image](https://user-images.githubusercontent.com/3626306/57927647-9159da80-78a6-11e9-8bb6-b0396d45b54f.png)

So in _this case_ I'd really like to remove the line breaks so these lines all wrap and the issue template doesn't look quite so long!